platforms/chibios/converters/elite_c_to_elite_pi/_pin_defs.h

```diff
@@ -18,7 +18,7 @@
 #define B5 9U
 
 // Right side (front)
-//      RAW
+#define B0 11U // back pad
```

Comment:
Likewise, I mapped this to 11U on elite_pi which is a pad accessible on the back of the microcontroller